Adds instructions on the use of AI (#1)
justeattakeaway/ui-coding-exercise: README.md

## ⚠️ Important considerations

> * Please do not use AI to generate the code. You may use AI bounce off ideas and ask general questions, but the solution must be your own original work.

Reviewer: ```suggestion
* Please do not use AI to generate your submission. Feel free to use AI to bounce ideas off and ask general questions, but the solution must be your own original work.
```
